ContextDesktop/src/main/java/com/github/migi_1/Context/Main.java

```diff
@@ -140,6 +140,22 @@ public void onAction(String name, boolean keyPressed, float tpf) {
                          default: //Do nothing when an unknown button is pressed.
                      }
                  }
+                 if(keyPressed){
+                        if(name.equals("steer_left")){
+                        environmentState.steer(-1.f);
+                    }
+                    if(name.equals("steer_right")){
+                        environmentState.steer(1.f);
+                    }
+                 }
+                 if(!(keyPressed)){
+                        if(name.equals("steer_left") || name.equals("steer_right")){
+                        environmentState.steer(0.f);
+                    }
+                    
+                 }
+                 
+                 
```

Comment:
Can you move this into a separate private method and then call that method here?
This initInputs method is long as it is.